boards/Kconfig

```
@@ -370,7 +370,16 @@ config ARCH_BOARD_ESP32S2_SAOLA_1
 	select ARCH_HAVE_IRQBUTTONS if ESP32S2_GPIO_IRQ
 	---help---
 		This is the ESP32-S2-Saola-1 board
-
+		
+config ARCH_BOARD_ESP32S2_FRANZININHO
+	bool "FRANZININHO WI-FI"
+	depends on ARCH_CHIP_ESP32S2WROVER
+	select ARCH_HAVE_LEDS
+	select ARCH_HAVE_BUTTONS
+	select ARCH_HAVE_IRQBUTTONS if ESP32S2_GPIO_IRQ
+	---help---
+		This is the ESP32-S2-FRANZININHO board
```

```suggestion
		Franzininho Wi-Fi is a development board to evaluate the ESP32-S2 SoC.
		https://docs.franzininho.com.br/docs/franzininho-wifi/franzininho-wifi/
```
Suggestion for a more detailed description.